src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -397,15 +396,7 @@ public virtual void ConfigureServices(IServiceCollection services)
             services.AddTransient<IReleasePermissionService, ReleasePermissionService>();
             services.AddTransient<IDataImportService, DataImportService>();
             services.AddTransient<IImportStatusBauService, ImportStatusBauService>();
-
-            services.AddTransient<IPublishingService, PublishingService>(provider =>
-                new PublishingService(
-                    provider.GetService<IPersistenceHelper<ContentDbContext>>(),
-                    new StorageQueueService(
-                        publisherStorageConnectionString,
-                        new StorageInstanceCreationUtil()),
-                    provider.GetService<IUserService>(),
-                    provider.GetRequiredService<ILogger<PublishingService>>()));
+            services.AddTransient<IPublishingService, PublishingService>();
```

Comment:
I thought we might want to use `AddSingleton` here? And with the others that use `QueueServiceClient`. 

I could be completely off here though...

Reply:
Added a new commit where I've changed the scope of our new 'clients' to Singleton. They wrap `QueueServiceClient` which in turn wraps `Azure.Storage.Queues.QueueServiceClient`.

The reason why I didn't set the scope to Singleton to begin with was that I wasn't sure if `QueueServiceClient` is thread-safe, and not using Singleton was just the safest option.

Looks like `Azure.Storage.Queues.QueueServiceClient` is thread-safe and we should be ok to make this change. Thanks.